robottelo/entities.py

```diff
@@ -2303,28 +2303,30 @@ class Meta(object):
         server_modes = ('sat', 'sam')
 
 
-class SyncPlan(orm.Entity, orm.EntityReadMixin, orm.EntityDeleteMixin,
+class SyncPlan(
+        orm.Entity, orm.EntityReadMixin, orm.EntityDeleteMixin,
         orm.EntityCreateMixin):
     """A representation of a Sync Plan entity."""
+    description = entity_fields.StringField()
     enabled = entity_fields.BooleanField(required=True)
-    name = entity_fields.StringField(required=True)
-    # how often synchronization should run must be one of: none, hourly, daily,
-    # weekly.
     interval = entity_fields.StringField(
         choices=('hourly', 'daily', 'weekly'),
         required=True,
     )
-    # start datetime of synchronization
+    name = entity_fields.StringField(required=True)
+    organization = entity_fields.OneToOneField('Organization', required=True)
     sync_date = entity_fields.DateTimeField(required=True)
-    description = entity_fields.StringField()
 
-    def __init__(self, organization_id, **kwargs):
-        """Record ``organization_id`` and set ``self.Meta.api_path``."""
-        self.Meta.organization_id = organization_id
+    def __init__(self, **kwargs):
```

Comment:
The first commit in this pull request contains an update to `OperatingSystemParameter.__init__`.